source/retryable-writes/tests/supported-single-statement.yml

```diff
@@ -24,7 +24,7 @@ tests:
         failPoint:
             times: 1
         operation:
-            name: "FindOneAndDelete"
+            name: "findOneAndDelete"
```

Comment:
I'm not sure if this actually affected anyone (perhaps folks were using `stricmp()`), but this is the canonical casing per the CRUD tests.

Reply:
I'll resolve this is a separate PR.